src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -1,60 +1,103 @@
+      if (accessible) {
+        availableServices.push(content);
+      } else {
+        unavailableServices.push(content);
+      }
+    });
+
+    return (
+      <div className="profile-section medium-12 columns">
+        <h4 className="section-header">Available services</h4>
         <div className="medium-12 columns">
-          <p><span className="label">Your account will allow you to:</span></p>
-          <p><a href="/health-care/apply/">Apply for health care</a></p>
-          <p><a href="/education/apply-for-education-benefits">Apply for education benefits</a></p>
-          <p><a href="/discharge-upgrade-instructions/">Apply for a discharge upgrade if needed</a></p>
-          <p><a href="/health-care/prescriptions">Refill your prescription</a></p>
-          <p><a href="/health-care/messaging">Message your health care team</a></p>
-          <p><a href="/health-care/health-records">Get your VA health records</a></p>
-          <p><a href="/track-claims">Check your claim and appeal status</a></p>
-          <p><a href="/education/gi-bill/post-9-11/ch-33-benefit">Get your Post-9/11 GI Bill statement of benefits</a></p>
-          {idCardAvailable && <p><a href="/veteran-id-card">Apply for a Veteran ID Card</a></p>}
+          {
+            !!availableServices.length && (
+              <p><span className="label">Your account will allow you to:</span></p>
+            )
+          }
+          {availableServices}
+          {
+            !!unavailableServices.length && (
+              <p><span className="label">You need to <a href="/verify?next=/profile">verify your identity</a> in order to:</span></p>
+            )
+          }
+          {unavailableServices}
           <p><strong>Note:</strong> If you have a less than honorable discharge, you can apply for an upgrade. If your
             application goes through and your discharge is upgraded, you'll be eligible for the VA benefits you earned
             during your period of service. <a href="/discharge-upgrade-instructions/">Find out how to apply for a discharge upgrade</a>.</p>
         </div>
-      );
-    } else {
-      content = (
-        <div className="medium-12 columns">
-          <p><span className="label">Your account will allow you to:</span></p>
-          <p><a href="/health-care/apply/">Apply for health care</a></p>
-          <p><a href="/education/apply-for-education-benefits">Apply for education benefits</a></p>
-          <p><a href="/discharge-upgrade-instructions/">Apply for a discharge upgrade if needed</a></p>
-          <p><span className="label">You need to <a href="/verify?next=/profile">verify your identity</a> in order to:</span></p>
-          <p>Refill your prescription</p>
-          <p>Message your health care team</p>
-          <p>Get your VA health records</p>
-          <p>Check your claim and appeal status</p>
-          <p>Check your Post-9/11 GI Bill statement of benefits</p>
-          <p>Apply for a Veteran ID Card</p>
```

Comment:
@patrickvinograd: Since the logic for showing this in the list is a little different now, just wanted to check if you thought it makes sense to display "Apply for a Veteran ID Card" under the verify list when `id-card` is not in the list of services in the user object.

Before, it wouldn't show up in that list as long as the user is LOA3, but the test user I was using was LOA3 and didn't have `id-card` in the service list, so it appears as shown in the screenshot in the description.

Reply:
Right now the frontend doesn't have enough information to make the claim it's making - because LOA3 vs. not is not the only reason why `id-card` doesn't show up in the services list. It could also be because they are ineligible based on title38 status. 

This is true for other services as well - but less likely to happen because most of the conditions for other services are more straightforward - e.g. "loa3 and has ssn". 

I think it would be more accurate to say something like "Verifying your account may give you access to more services:" with a list, and maybe additionally a tool tip or something describing what the requirements are for each unavailable service. That would be a decision to be made by UX and/or PM though.

Reply:
I think the context of this view is meant to be precisely that. By identity proofing you could potentially use these additional features, but not that you absolutely can. If we need to change the language of the "Available Services" section to more accurately reflect that I think we should. 

But keeping the logic behind this on the front-end somewhat streamlined makes the most sense to me, especially since a lot of those other attributes are only ever available once we're able to query MVI (LOA3 only).